README.md

```diff
@@ -1,2 +1,72 @@
-# cloud-action
-Connect your GitHub repo to Grouparoo Cloud
+# grouparoo/cloud-action
+
+A GitHub Action for connecting your GitHub repository to Grouparoo Cloud.
+
+To use this Action you'll need to have an existing [Grouparoo Cloud](https://cloud.grouparoo.com/) project. If you don't have one yet, you can sign up for a free trial at [grouparoo.com/trial](https://grouparoo.com/trial).
+
+## Usage
+
+Make sure to first checkout the repository by using [actions/checkout](https://github.com/marketplace/actions/checkout). Then, this action will do the rest.
+
+```yml
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: grouparoo/cloud-action@v1
```

Comment:
We haven't published this action yet, but my goal is to do so after this README is in. However, I'm not sure on version numbers - since Grouparoo itself is not yet 1.0.0, does it make sense that this is? Or should we start with 0.1.0?